frontend/src/mixins/requests/UserInfoApiRequestMixin.ts

```diff
@@ -0,0 +1,40 @@
+import RequestUtils from "@/utils/RequestUtils";
+import { Component, Mixins } from "vue-property-decorator";
+import ErrorHandler from "./ErrorHandler";
+import { Userinfo } from "@/types/common/Userinfo";
+
+@Component
+export default class UserInfoApiRequestMixin extends Mixins(ErrorHandler) {
+  async getUserinfo(): Promise<Userinfo> {
+    const fetchServicesUrl = import.meta.env.VITE_VUE_APP_API_URL + "/api/sso/userinfo";
+
+    const userinfo: Userinfo = await fetch(fetchServicesUrl, RequestUtils.getGETConfig())
+      .then((response) => {
+        return response.json();
+      })
+      .then((json) => {
+        return this.mapJson(json);
+      })
+      .catch((error) => {
+        throw this.handleError(true, error);
```

Comment:
Da die Daten in der Komponente `App.vue` initial geholt werden, muss die Fehlermeldung als Toast angezeigt werden.

Zusätzlich Testen ob und wie die Fehlermeldung angezeigt wird, da das Backend mit einer fest definierten Art von Fehler antwortet.

Der Keycloak wird definitiv anders bzw. mit einer anderen Fehlerresponse im Body regaieren. Evtl sogar `throw this.handleError(true, error);` durch einen allgemeinen Toast ersetzen (z.B. Nutzerinformationen konnten nicht gelade nwerden).